tools/gstf_preparation/gstf_preparation.py

```diff
@@ -92,6 +92,12 @@ def create_tables(conn):
         FROM transcript JOIN gene
         USING (gene_id)''')
 
+    cur.execute('''CREATE TABLE sytenic_region (
```

Comment:
I suppose that's a typo?
```suggestion
    cur.execute('''CREATE TABLE syntenic_region (
```

There's other 9 instances of the same typo.